.github/workflows/apply-migrations.yaml

```diff
@@ -33,23 +33,14 @@ jobs:
       - name: Install dependencies
         run: npm install
         
-      - name: Generate migration preview
-        run: npm run db:generate
-        env:
-          POSTGRES_URL: ${{ secrets.POSTGRES_URL }}
-          
       - name: Run migration preview
-        run: |
-          npm run run-migrations -- --preview > migration-preview-full.txt 2>&1
-          sed -n '/=== Generated migration files ===/,$p' migration-preview-full.txt > migration-preview.txt
+        run: npm run run-migrations -- --preview
         env:
           POSTGRES_URL: ${{ secrets.POSTGRES_URL }}
```

Comment:
set this as `POSTGRES_URL: ${{ secrets.NEON_URL }}` instead to see if that works